pandas/core/arrays/sparse/array.py

```diff
@@ -708,6 +708,18 @@ def npoints(self) -> int:
         """
         The number of non- ``fill_value`` points.
 
+        This property returns the number of elements in the sparse series that are
+        not equal to the ``fill_value``. Sparse data structures store only the
+        non-``fill_value`` elements, reducing memory usage when the majority of
+        values are the same.
+
+        See Also
+        --------
+        Series.sparse.to_dense : Convert a Series from sparse values to dense.
+        Series.sparse.fill_value : Elements in `data` that are `fill_value` are
+            not stored.
```

Comment:
```suggestion
        Series.sparse.fill_value : Elements in ``data`` that are ``fill_value`` are
            not stored.
```

Reply:
@mroeschke do you think double tick is needed here? I want to understand where to use single tick vs double tick. Appreciate your explanation in advance!

Reply:
Yeah I think double ticks would be good here. Generally I think arguments to methods/functions or any Python identifiers should use double ticks